browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -4006,6 +4006,9 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_UPDATE_FILTER_LIST_FAIL_TEXT" desc="Text to notify users for the failure of updating filterlist">
       Filter list could not be updated. Please try again
     </message>
+    <message name="IDS_SHOW_UNDO_ON_TAB_CLOSED" desc="Menu item to show undo button on tab closed">
+      Show undo button on tab closed
```

Comment:
@timchilds are we ok with that string?

Reply:
Yeah, @timchilds agreed. `Show undo button when tab is closed` is better

Also, will this be only a tab-level or site-level option? If not (i.e. if it's a more general setting that applies on any tab or site) then I wonder if pluralizing would make even more sense? So like `Show unto button when tabs are closed`

Reply:
@rmcfadden3 will be for all tabs closed - so once disabled, person won't see the undo button anymore

Reply:
Done!